ledger-core/v2/application/testwalletapi/testwalletserver.go

```diff
@@ -29,7 +29,9 @@ type TestWalletServer struct {
 }
 
 const (
-	create = "New"
+	create     = "New"
```

Comment:
Give comment and rename this constant. It's not clear, what there are about

Reply:
Is it more clear for you?
// Aliases VCallRequest.CallSiteMethod for call testwallet.Wallet contract methods
const (
	create     = "New"     // Create wallet
	getBalance = "Balance" // Get wallet balance
	addAmount  = "Accept"  // Add money to wallet
)